### PR TITLE
Feature/iops 1626 healthcare service new

### PR DESCRIPTION
**Pull request by SheenaSharma**

### CODE REVIEW

File: CapabilityStatement/England-CapabilityStatement.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<CapabilityStatement xmlns="http://hl7.org/fhir">
+    <id value="England-CapabilityStatement" />
+    <url value="https://fhir.nhs.uk/StructureDefinition/England-CapabilityStatement" />
+    <version value="1.0.0" />
+    <name value="EnglandCapabilityStatement" />
+    <title value="England CapabilityStatement" />
+    <status value="draft" />
+    <date value="2023-06-01" />
+    <publisher value="NHS England" />
+	<contact>
```
Comment (SBajanki1) [inline CapabilityStatement/England-CapabilityStatement.xml:11]: Needs alignment

File: CapabilityStatement/England-CapabilityStatement.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      <name value="NHS England" />
+      <telecom>
+        <system value="email" />
+        <value value="interoperabilityteam@nhs.net" />
+        <use value="work" />
+        <rank value="1" />
+      </telecom>
+	</contact>
+    <description value="Conformance requirements for NHS England Implementation Guide" />
```
Comment (SBajanki1) [inline CapabilityStatement/England-CapabilityStatement.xml:20]: Not sure with description

File: codesystems/CodeSystem-England-ERSAppointmentType.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    <status value="draft" />
+    <date value="2023-06-01" />
+    <publisher value="NHS England" />
+    <contact>
+        <name value="NHS England" />
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
```
Comment (SBajanki1) [inline codesystems/CodeSystem-England-ERSAppointmentType.xml:16]: check alignment

File: StructureDefinition/England-HealthcareService.xml [hunk 4]
```diff
@@ -0,0 +1,320 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+    <id value="England-HealthcareService" />
+    <url value="https://fhir.nhs.uk/StructureDefinition/England-HealthcareService" />
+    <name value="EnglandHealthcareService" />
+	<title value="England HealthcareService" />
+    <status value="draft" />
```
Comment (SBajanki1) [inline StructureDefinition/England-HealthcareService.xml:6]: Date is missing

File: codesystems/CodeSystem-England-ERSPurpose.xml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    <status value="draft" />
+    <date value="2023-06-01" />
+    <publisher value="NHS England" />
+    <contact>
+        <name value="NHS England" />
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+			<rank value="1" />
```
Comment (SBajanki1) [inline codesystems/CodeSystem-England-ERSPurpose.xml:16]: check line alignment

File: codesystems/CodeSystem-England-ERSSpecialty.xml [hunk 6]
```diff
@@ -0,0 +1,387 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="England-ERSSpecialty" />
+    <url value="https://fhir.nhs.uk/CodeSystem/England-ERSSpecialty" />
+    <version value="1.0.0" />
+    <name value="EnglandERSSpecialty" />
+	<title value="England ERS Speciality" />
+    <status value="draft" />
```
Comment (SBajanki1) [inline codesystems/CodeSystem-England-ERSSpecialty.xml:7]: Date is missing

File: valusets/ValueSet-England-ERSClinicType.xml [hunk 7]
```diff
@@ -0,0 +1,26 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="England-ERSClinicType" />
+    <url value="https://fhir.nhs.uk/ValueSet/England-ERSClinicType" />
+    <version value="1.0.0" />
+    <name value="EnglandERSClinicType" />
+	<title value="England ERS Clinic Type" />
+    <status value="draft" />
```
Comment (SBajanki1) [inline valusets/ValueSet-England-ERSClinicType.xml:7]: Date is missing

File: valusets/ValueSet-England-ERSSpeciality.xml [hunk 8]
```diff
@@ -0,0 +1,26 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="England-ERSSpeciality" />
+    <url value="https://fhir.nhs.uk/ValueSet/England-ERSSpecialty" />
+    <version value="1.0.0" />
+    <name value="EnglandERSSpecialty" />
+	<title value="England ERS Speciality" />
+    <status value="draft" />
```
Comment (SBajanki1) [inline valusets/ValueSet-England-ERSSpeciality.xml:7]: Date is missing

**Review by SBajanki1 — CHANGES_REQUESTED** [review]
1. Added comments
2. Check line alignments for all the files

File: valusets/ValueSet-England-ERSClinicType.xml (hunk 7 above)
Comment (SBajanki1) [reply valusets/ValueSet-England-ERSClinicType.xml:7]: No need to change in the feature branch, it already exists in develop branch

File: valusets/ValueSet-England-ERSSpeciality.xml (hunk 8 above)
Comment (SBajanki1) [reply valusets/ValueSet-England-ERSSpeciality.xml:7]: No need to change in the feature branch, it already exists in develop branch

File: CapabilityStatement/England-CapabilityStatement.xml (hunk 1 above)
Comment (nhse-mf) [reply CapabilityStatement/England-CapabilityStatement.xml:11]: Will be dealt with when merging in from develop

**Comment by nhse-mf** [comment]
Fixes made based on above comments

**Review by SBajanki1 — APPROVED** [review]

**Review by SBajanki1 — APPROVED** [review]

**Review by KazeemHamzat — DISMISSED** [review]

**Review by KazeemHamzat — APPROVED** [review]

**Review by SBajanki1 — APPROVED** [review]

**Pull request merged**